.idea/GitAssemblyUtil.iml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why do I not see changes in `kbase.yml` but I do see changes in the compiled server?

Reply:
I don't know...

Reply:
Is your kbase.yml the same as in github?

Reply:
Yes.

Reply:
Ok, I see. The version in master is 1.1.0, so I'm presuming no one has recompiled the server since 1.0.7.  Since you recompiled the server 1.1.0 shows up with no change to the yml file.

You do need to bump the version though.

Reply:
added version 1.2.0.

Reply:
You say 1.2.0 here but in kbase.yml it's 1.1.1. I think 1.2.0 is right fwiw